lib/identity_config.rb

```diff
@@ -399,6 +399,7 @@ def self.store
     config.add(:socure_document_request_endpoint, type: :string)
     config.add(:socure_idplus_api_key, type: :string)
     config.add(:socure_idplus_base_url, type: :string)
+    config.add(:socure_idplus_endpoint, type: :string)
```

Comment:
looks like team Ada has gotten here first and added `socure_idplus_base_url` .... perhaps we should coalesce and perhaps adopt their approach 🤔

Reply:
Done in a66eed5